react-events/my-app/src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import { Header } from './Header';
+import { Image } from './Image';
+import { Caption } from './Caption';
+import { Description } from './Description';
+import { Button } from './Button';
+import { useState } from 'react';
+import './App.css';
+
+const srcs = ['/starry-sky.jpeg', '/cool-kitty.jpg', '/cool-car.jpg'];
```

Comment:
This will work, but there is a better way. I gave a hint during the Q&A. We can use an array of imageInfo objects, each with an src, caption, and description attribute. You can then pass the prop as such `imageInfo[index].caption`